py-polars/polars/_typing.py

```diff
@@ -315,6 +315,8 @@ def fetchmany(self, *args: Any, **kwargs: Any) -> Any:
     Literal["auto", "in-memory", "streaming", "gpu"], "GPUEngine"
 ]
 
+PlanStage: TypeAlias = Literal["ir", "phys"]
```

Comment:
The Polars public API strongly prefers full names. So this should be `"ir", "physical"`.